content/dstu2/related-person.md

```diff
@@ -31,18 +31,18 @@ Notes:
 
 ### Response
 
-<%= headers 200, GET: '[...]/RelatedPerson?identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:IDENTITY-FEDERATION:REALM:AC193C1A-9763-45E1-9FA2-C1C3AA4BA16D:PRINCIPAL:PI98N2FK5TN' %>
+<%= headers 200, GET: '[...]/RelatedPerson?_id=5796348' %>
```

Comment:
@cerbs would we want to keep this example as an identifier search or is it ok to switch to search by _id?